apps/aecore/src/aecore.app.src

```diff
@@ -34,6 +34,7 @@
     unicode_util_compat, %% to be removed, not needed in OTP 20
     idna,
     aecuckoo,
```

Comment:
Why do we keep the old miners ?

Reply:
Yes.

This enables user to configure prebuilt miner while deferring solving the problem of identifying which OS needs to be selected.